.teamcity/src/main/kotlin/model/FunctionalTestBucketProvider.kt

```diff
@@ -153,7 +153,7 @@ class StatisticBasedFunctionalTestBucketProvider(private val model: CIBuildModel
     }
 
     private fun determineSubProjectClassTimes(testCoverage: TestCoverage, buildProjectClassTimes: BuildProjectToSubprojectTestClassTimes): Map<String, List<TestClassTime>>? {
-        val testCoverageId = testCoverage.asId("Gradle_Check")
+        val testCoverageId = testCoverage.asId("Gradle_Master_Check")
```

Comment:
Can you extract the configuration name to a constant?

Reply:
Followed up here: https://github.com/gradle/gradle/pull/17505